src/gleam/string.gleam

```diff
@@ -258,9 +316,51 @@ pub fn join(strings: List(String), with separator: String) -> String {
 /// pad_left("121", to: 2, with: ".") == "121"
 /// ```
 ///
-// pub fn pad_left(string: String, to size: Int, with: String) {}
+type PadSide {
```

Comment:
The doc comment above is attached to this private type rather than the function it documents